src/Server/TCPHandler.cpp

```diff
@@ -1313,6 +1330,14 @@ bool TCPHandler::isQueryCancelled()
 
 void TCPHandler::sendData(const Block & block)
 {
+    /// For testing hedged requests
```

Comment:
Probably we can just run a query like
SELECT sleep(x) and have different x stored in table on different nodes of distributed table? We even can disable replication and insert different rows in replicated table